bleak/backends/bluezdbus/client.py

```diff
@@ -382,9 +384,7 @@ async def _remove_signal_handlers(self) -> None:
 
         self._bus.remove_message_handler(self._parse_msg)
 
-        # avoid reentrancy issues by taking a copy of self._rules
-        old_rules, self._rules = self._rules, []
-        for rule in old_rules:
+        for rule in self._rules:
```

Comment:
I think we still need `self._rules = []` (or `self._rules.clear()`) after this loop.

Reply:
Indeed